sklearn/metrics/_ranking.py

```diff
@@ -538,6 +538,21 @@ class scores must correspond to the order of ``labels``,
     RocCurveDisplay.from_predictions : Plot Receiver Operating Characteristic
         (ROC) curve given the true and predicted values.
 
+    Notes
+    -----
+    The Gini Coefficient is a summary measure of the ranking ability of binary
+    classifiers. It is expressed using the area under of the ROC as follows:
+
+    G = 2 * AUC - 1
+
+    Where G is the Gini coefficient and AUC is the ROC-AUC score. This normalisation
+    will ensure that random guessing will yield a score of 0 in expectation, and it is
+    upper bounded by 1.
+
+    Note that there is another version of the Gini coefficient for regressors of a
+    continuous positive target variable. In this case, AUC is taken over the Lorenz
+    curve instead of the ROC [6]_.
```

Comment:
This is not entirely correct. Maybe best to just delete these 2 sentences:
- There is a difference between the Gini index as a property of a probability distribution (as used in economics) and the Gini index as used in ML.
- Whether for binary classification or regression, the ML Gini index is some ratio of areas under the Cumulative Accuracy Profile (CAP), aka gain curve or lift curve.
- If we want to add anything here, I would rather add that the Gini index (for binary classification), Wilcoxon–
Mann–Whitney U -statistics and Somers' D statistic all describe the same quantity, up to a scaling factor.

For details see https://ssrn.com/abstract=4248143.

Reply:
Alright, let's do a follow-up PR to delete those last two sentences to avoid introducing additional confusion.

Reply:
@ogrisel @lorentzenchr hi there! It was addition provided by @ogrisel, so to be fairly I haven’t checked it in a proper way. In several hours I’ll fix this with a new PR. Thanks for the response.

Reply:
No need to apologize. And thanks for the quick fix.